storage/blackhole/ha_blackhole.h

```diff
@@ -68,9 +68,9 @@ class ha_blackhole: public handler
             HA_READ_ORDER | HA_KEYREAD_ONLY);
   }
   /* The following defines can be increased if necessary */
-#define BLACKHOLE_MAX_KEY	64		/* Max allowed keys */
+#define BLACKHOLE_MAX_KEY	MAX_KEY		/* Max allowed keys */
 #define BLACKHOLE_MAX_KEY_SEG	16		/* Max segments for key */
-#define BLACKHOLE_MAX_KEY_LENGTH 1000
+#define BLACKHOLE_MAX_KEY_LENGTH 3500 /* Like in innodb */
```

Comment:
Align the comment with ones above, s/innodb/InnoDB

Reply:
Aligned.